sdk/cosmos/examples/trigger_00.rs

```diff
@@ -62,20 +62,14 @@ async fn main() -> Result<(), Box<dyn Error + Send + Sync>> {
 
     let ret = trigger_client
         .create_trigger()
-        .trigger_type(TriggerType::Post)
-        .trigger_operation(TriggerOperation::All)
-        .body(&"something")
-        .execute()
+        .execute("something", TriggerType::Post, TriggerOperation::All)
```

Comment:
We might want to make trigger type and trigger operation optional for users to set (even though the API requires sending them), and just default to `Post` and `All`